trulens_eval/examples/snowflake_demo/llm.py

```diff
@@ -109,6 +116,14 @@ def _generate_stream_with_replicate(
         for t in stream_iter:
             yield str(t)
 
+    def _generate_with_cortex(self, model_name: str, model_input: dict):
+        response = get_provider("Cortex")._create_chat_completion(
+            prompt=model_input['prompt'],
+            temperature=model_input['temperature'],
+            model=model_name
+        )
+        yield str(response)
```

Comment:
This still isn't streaming-when-first-token-is-available right? (or perhaps something changes over the last few days?)

Reply:
Not streaming, but made this method a generator to match replicate. It will yield the full response just once